apps/vr-tests/package.json

```diff
@@ -5,11 +5,12 @@
   "description": "Visual regression tests for Fluent UI React",
   "scripts": {
     "build": "just-scripts build",
-    "clean": "",
+    "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "screener": "screener-storybook --conf screener.config.js",
+    "screener": "screener-storybook --conf screener.config.js --debug",
```

Comment:
is the --debug intentional? I am assuming the "screener" task is run in CI.

Reply:
Yes, but it's been unreliable, and enabling this option may provide a little more info to help debug failures.

Reply:
Unfortunately, we need `--debug` for the error stack (see [#20210305.8](https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=192933&view=logs&j=a8a82d6e-f5e1-5356-31e4-5989f531914b&t=a9a7a2c7-74f0-584a-6e7d-b8881e9dc148&l=1460)). Either that or we have a log truncating issue